Cython/Compiler/Code.py

```diff
@@ -1138,7 +1138,8 @@ class GlobalState(object):
         'pystring_table',
         'cached_builtins',
         'cached_constants',
-        'init_globals',
+        'init_globals',  # init_globals generates an init_globals_body insertion point
+            # that comes before the utility code
```

Comment:
Why not use separate functions (and sections) for these? E.g. `InitGlobals`, `InitStrings`, `InitConstants`, or something along those lines. There isn't really a reason why they should all share the same section, except for laziness. And that now gets in the way.

Reply:
Right - I've split this into two `init_globals` (defines function `__Pyx_InitGlobals`) and `init_constants` (defines function `__Pyx_InitConstants`). The main reason I didn't split it into three with a separate `InitStrings` function is that there was already a `__Pyx_InitStrings` function, and I was struggling to come up with a clear name to distinguish them.

That's still slightly lazy, but I think the main thing is that the utility code has its own section.